.github/workflows/ctc.yml

```diff
@@ -18,8 +18,8 @@ jobs:
           -H "Accept: application/json" \
           -H "Content-Type: application/json" \
           -H "Authorization: Token ${{ secrets.TPS_API_TOKEN_PARAM }}" \
-          -d '{"lock": {"sha": "${{ github.sha }}", "component_name": "${{ secrets.TPS_API_APP_ID }}"}}' \
-          ${{ secrets.TPS_API_URL_PARAM }}/api/ctc`
```

Comment:
I see you removed the path `/api/ctc` here. Was that moved to the URL set in the secrets? In that case I believe you need to do the same for retries on line 45 below because it still has that path set.

Reply:
Fixed for now by hardcoding the TPS URL, and I'm asking Prod Eng about what `TPS_API_URL_PARAM`  supposedly contains.